startup/20-motors.py

```diff
@@ -99,7 +99,11 @@ class HHM(Device):
 
     def __init__(self, *args, enc = None, **kwargs):
         super().__init__(*args, **kwargs)
-        self.pulses_per_deg = 1/self.main_motor_res.value
+        try:
+            self.pulses_per_deg = 1 / self.main_motor_res.get()
+        except ZeroDivisionError:
+            self.pulses_per_deg = -1
```

Comment:
@elistavitski, is it a reasonable approach? Or what do you want to do in this case?